src/navs/index.js

```diff
@@ -1494,6 +1494,10 @@ export const docsMenu = {
                             name: 'Users',
                             url: '/docs/api/user',
                         },
+                        {
+                            name: 'Query',
+                            url: '/docs/api/query',
```

Comment:
This file doesn't exist... or what am I missing?

Reply:
https://posthog.com/docs/api/query - it automatically gets generated by the openapi schema